src/App.js

```diff
@@ -1,24 +1,54 @@
-import React from 'react';
-import { Switch, Route } from 'react-router-dom';
+import React, { useState } from 'react';
+import { Switch, Route, Redirect, Link } from 'react-router-dom';
 
-import { UserMain } from './containers/userMain/UserMain';
-import { Login } from './containers/login/Login';
 import { Appointments } from './containers/appointments/Appointments';
-import { Navbar } from './components/navbar/Navbar';
+import { Contacts } from './containers/contacts/Contacts';
 
 function App() {
+  const [appointments, setAppointments] = useState([]);
+  const [contacts, setContacts] = useState([]);
+
+  const addAppointment = (title, contact, date) => {
+    setAppointments([
+      ...appointments, 
+      {
+        title: title, 
+        contact: contact, 
+        date: date.toLocaleDateString('en-US')
+      }
+    ]);
+  };
+
+  const addContact = (name, phone, email) => {
+    setContacts([
+      ...contacts, 
+      {
+        name: name, 
+        phone: phone, 
+        email: email}
+    ]);
+  };
+
   return (
-    <div>
-      <p>App State: NavbarInfo (App Logo or Patient Info or Provider Info)</p>
-      <Navbar />
+    <>
+      <nav>
+        <Link to="/appointments">Appointments</Link>
+        <Link to="/contacts">Contacts</Link>
+      </nav>
       <Switch>
-        <Route exact path="/"><UserMain /></Route>
-        <Route path="/login"><Login /></Route>
-        <Route exact path="/provider"><Appointments userType="Provider"/></Route>
-        <Route exact path="/patient"><Appointments userType="Patient"/></Route>
+        <Route exact path="/"><Redirect to="/appointments" /></Route>
+        <Route path="/contacts"><Contacts 
+          contacts={contacts} 
+          addContact={addContact} 
+        /></Route>
```

Comment:
Prettier will fix this, but I'm flagging here just in case: The `<Route>` and `<Contacts>` components should be on separate lines. This is the conventional styling and it helps learners differentiate what they need to edit vs. leave alone.

Reply:
Done.